sources/packages/backend/libs/test-utils/src/factories/application.ts

```diff
@@ -154,3 +156,29 @@ export async function saveFakeApplicationDisbursements(
   savedApplication.currentAssessment = savedOriginalAssessment;
   return applicationRepo.save(savedApplication);
 }
+
+/**
+ * Create and save a fake application.
+ * @param dataSource data source to persist application.
+ * @param relations application entity relations.
+ * - `student` related student.
+ * - `programYear` related program year.
+ * - `currentStudentAssessment` related current assessment.
+ * - `applicationException` related application exception.
+ * - `institutionLocation` related institution location.
+ * @returns persisted application with relations provided.
+ */
+export async function saveFakeApplication(
```

Comment:
I miss this in the first round of the review, what is the motivation to create this method?
My assumption is that the `saveFakeSimething` would be used when we have a combination of items that must be saved altogether and there is a benefit to grouping those saving in one method. Am I missing something?

Reply:
I got it. Can use save at test case level.